test/unittest/tests/test_session_config.cpp

```diff
@@ -0,0 +1,66 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
```

Comment:
Add a comment at the start of the file stating what the test is testing.

Reply:
838a57d7b

Reply:
Thank you.